reclass/config.py

```diff
@@ -32,6 +32,9 @@ def make_db_options_group(parser, defaults={}):
     ret.add_option('-z', '--ignore-class-notfound', dest='ignore_class_notfound',
                    default=defaults.get('ignore_class_notfound', OPT_IGNORE_CLASS_NOTFOUND),
                    help='decision for not found classes [%default]')
+    ret.add_option('-a', '--add-subdir-to-node', dest='add_subdir_to_node', action="store_true",
+                   default=defaults.get('full_node_path', OPT_ADD_SUBDIR_TO_NODE),
+                   help='Add subdir when generating node names. [%default]')
```

Comment:
I'm happy to change the long name `add_subdir_to_node`, short name or description to anything else.